src/main/java/science/icebreaker/account/Account.java

```diff
@@ -0,0 +1,91 @@
+package science.icebreaker.account;
+
+import io.swagger.annotations.ApiModelProperty;
+import org.springframework.lang.Nullable;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.security.Principal;
+import java.util.Objects;
+
+
+@Entity
+public class Account implements Principal {
+
+    @Id
+    @Nullable
```

Comment:
Because a new account that should be created does not have an ID. It would also be possible to differentiate between something like `AccountLoginData` (username, password) and `Account` (id, username, password) with the `AccountLoginData` then being used as the request parameter in the AccountController. I thought that this would be a little too complex, what do you think?